src/cgame/cgame.h

```diff
@@ -499,6 +521,44 @@ typedef struct cg_import_s {
 	 * @remarks This function never returns `NULL`, but instead will return the null texture.
 	 */
 	r_image_t *(*LoadImage)(const char *name, r_image_type_t type);
+	
+	/**
+	 * @brief Creates a blank state for an atlas and returns it.
+	 *
+	 * @param name The name to give to the atlas, e.g. `"my_atlas_is_cool"`
+	 *
+	 * @return The atlas that has been created.
+	 *
+	 * @remarks Start with this function, add images to it with AddImageToAtlas, then call StitchAtlas when
+	 * you are ready. You can then query positions with GetAtlasImageFromAtlas.
+	 */
+	r_atlas_t *(*CreateAtlas)(const char *name);
+
+	/**
+	 * @brief Add an image to the list of images for this atlas.
+	 *
+	 * @param atlas The atlas to add an image to.
+	 * @param image The image to add to the atlas.
+	 */
+	void (*AddImageToAtlas)(r_atlas_t *atlas, const r_image_t *image);
+	
+	/**
+	 * @brief Resolve an atlas image from an atlas and image.
+	 *
+	 * @param atlas The atlas to fetch the stitched image from.
+	 * @param image The original image you wish to query.
+	 *
+	 * @return An r_atlas_image_t which contains the stitched coordinates of that image.
+	 */
+	const r_atlas_image_t *(*GetAtlasImageFromAtlas)(const r_atlas_t *atlas, const r_image_t *image);
+
+	/**
+	 * @brief Stitches the images together into an atlas.
+	 *
+	 * @param atlas The atlas to stitch together.
+	 */
+	void (*StitchAtlas)(r_atlas_t *atlas);
+
```

Comment:
Stitch is kind of a weird word. CompileAtlas or FinalizeAtlas or something maybe?